woocommerce/woocommerce-gutenberg-products-block
# Product Gallery: CSS styling tightening up

## What

The goal of this PR is to ensure the `Product Gallery` block displays correctly in all setting configurations and is not dependent on the neighboring blocks.

Fixes #10838 https://github.com/woocommerce/woocommerce-blocks/issues/10750

## Why

This PR attempts to rectify styling shortcomings that arose from various `Product Gallery` settings being added and adding complexity.

## Testing Instructions

1. Using a block theme, go to `Appearance > Editor`.
2. Go to `Templates > Single Product`.
3. Make sure you are using the blockified template.
4. Add the **Product Gallery** block (replacing the legacy `Product Image Gallery` block).
5. _After completing the below steps, go back to step 4. and insert the `Product Gallery` block just above `Product Details` tabs, so that it doesn't have any neighboring columns._
6. Verify that the `Product Gallery` displays correctly (doesn't overflow etc.) in all possible configurations of the `Thumbnails` Positon setting and the `Next/Prev Buttons` appearance setting, both in the editor and frontend.




## Screenshots or screencast



| Before | After |
| ------ | ----- |
|<img width="1249" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/6a7467ab-af31-4b4d-804d-c28a29a67f9c">|<img width="973" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/4b67f2ae-7bc0-4c65-92fd-e570f7ae75e0">|


| Before | After |
| ------ | ----- |
|<img width="1041" alt="V-Neck_T-Shirt_–_newproductgallery-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/8d3d47bd-1ec9-4c76-9862-a13f1cd13dd0">|<img width="1000" alt="V-Neck_T-Shirt_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/0d855e0a-e1c1-458c-9550-3161777f07d8">|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Product Gallery: Fix display issues in the editor and the page.
